src/lib/browse/layers/points/Problems.svelte

```diff
@@ -0,0 +1,114 @@
+<script lang="ts">
+  import {
+    HelpButton,
+    Popup,
+    privateResourceBaseUrl,
+    Legend,
+  } from "lib/common";
+  import { Checkbox } from "govuk-svelte";
+  import { layerId, constructMatchExpression } from "lib/maplibre";
+  import { CircleLayer, GeoJSON } from "svelte-maplibre";
+  import { showHideLayer } from "../url";
+
+  let name = "problems";
+
+  let show = showHideLayer(name);
+
+  // Display full details in a certain order
+  let keys = [
+    "Issue ID",
+    "Scheme ID",
+    "Date Logged",
+    "Inspector",
+
+    "Street Location",
+    "Code",
+
+    "Issue Description",
+    "External Inspectorate Comment for LA",
+
+    "Issue Stage",
+    "Resolved by Design",
+    "Design Review Status",
+    "Date of Resolution",
+    "Design Resolution",
+  ];
+
+  let legend: [string, string][] = [
+    ["Outstanding", "red"],
+    ["Pending", "yellow"],
+    ["Resolved", "green"],
+    ["Not Applicable", "grey"],
+    ["No Resolution Planned", "black"],
+  ];
+</script>
+
+<Checkbox bind:checked={$show}>
+  Issues & Conflicts
+  <span slot="right">
+    <HelpButton>
+      <p>
+        Critical issues and policy conflicts from the DA Log. This is internally
```

Comment:
maybe call it Design Assistance Log just for absolute clarity, I can think of a few possible reasonable answers (Assurance, Aid, Assessment)